### PR TITLE
Timber Wolf - updated keymaps, fixed info.json

### PR DESCRIPTION
**Pull request by Croktopus**
## Description

Primary goal is to fix the info.json file, as an issue caused the layouts to show improperly in QMK Configurator - I believe I identified and fixed the problem (fingers crossed) which i think was caused by Y values that were .25 off or what have you being in the wrong row.

Also made small changes to some keymaps, switched to hardware pwm, and made more preparations for VIA

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by Croktopus** [comment]
Yeah, since the keys overlap on the pcb it seemed right and proper. hopefully it's alright, though if necessary i can nudge them over

File: keyboards/metamechs/timberwolf/config.h [hunk 1]
```diff
@@ -24,8 +24,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID 0x5754 // TW
 #define DEVICE_VER 0x0001
 #define MANUFACTURER MetaMechs
-#define PRODUCT Timber Wolf
-#define DESCRIPTION Timber Wolf
+#define PRODUCT MetaMechs Timber Wolf
+#define DESCRIPTION MetaMechs Timber Wolf
```
Comment (fauxpark) [inline keyboards/metamechs/timberwolf/config.h:28]:
```suggestion
#define PRODUCT Timber Wolf
```

**Comment by noroadsleft** [comment]
> Yeah, since the keys overlap on the pcb it seemed right and proper. hopefully it's alright, though if necessary i can nudge them over

For `LAYOUT_all`, I'd prefer to have the highlighted section nudged over so it doesn't overlap.

image

Also, the key that's `KC_PENT` in the highlight also has a key beneath it at the same coordinates (two keys with the same `x` and `y` values). I'd like that fixed.

`LAYOUT_all` is [explicitly not required to be a layout that is possible to build](https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=ltkeyboard_namehgt) (admittedly, this is a very obscure section of the docs):

> When defining multiple layouts you should have a base layout, named `LAYOUT_all`, that supports all possible switch positions on your matrix, even if that layout is impossible to build physically.

File: keyboards/metamechs/timberwolf/config.h (hunk 1 above)
Comment (Croktopus) [reply keyboards/metamechs/timberwolf/config.h:28]: I was trying to go by the VIA standard here: https://caniusevia.com/docs/configuring_qmk#change-product
is that outdated?

**Comment by Croktopus** [comment]
thanks for the feedback - I've made some changes, but it's more than a little sloppy looking. Is it legal to use keys that are like .5 or .75 width?

**Comment by Croktopus** [comment]
People are gonna start getting their keyboards in a few days, anyone available to give it a final review and merge?

File: keyboards/metamechs/timberwolf/config.h (hunk 1 above)
Comment (fauxpark) [reply keyboards/metamechs/timberwolf/config.h:28]: I disagree with the recommendation there, as the `MANUFACTURER` string is where you should put your name, and `PRODUCT` where you put the name of the keyboard. The fact that some OSes don't show both is an issue with them not accommodating the USB spec (although admittedly it doesn't say much about the manu/product strings).

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**